src/routes/quickstart/+page.ts

```diff
@@ -0,0 +1,19 @@
+import { error } from '@sveltejs/kit';
+import { ERROR_TYPE } from '$lib/constants/ErrorTypes';
+
+// all static mdx files must be pre-rendered at build time instead of default ssr generated
+export const prerender = true;
+
+export async function load() {
+  const docs = await import('../../quickstart/_main.mdx').catch((err) => {
```

Comment:
I get a TypeScript error locally. Not sure if you have some config I don’t or maybe there’s just an error here we need to ignore.

Reply:
Good catch. I've added a `mdx.d.ts` file (as per GH Copilot's suggestion) and that's fixed the error